src/main/scala/com/fulcrumgenomics/util/RefFlatSource.scala

```diff
@@ -122,13 +123,13 @@ class RefFlatSource private(lines: Iterator[String],
     var numDifferentStrands     = 0
     var numTranscripts          = 0
     val _genes = new DelimitedDataParser(lines=_lines, delimiter='\t').flatMap { row =>
-      val geneName       = row[String]("geneName")
-      val transcriptName = row[String]("name")
-      val contig         = row[String]("chrom")
-      val strand         = row[String]("strand")
-      val exonCount      = row[Int]("exonCount")
-      val exonStarts     = row[String]("exonStarts").split(',').filter(_.nonEmpty).map(_.toInt)
-      val exonEnds       = row[String]("exonEnds").split(',').filter(_.nonEmpty).map(_.toInt)
+      val geneName       = row.string("geneName")
```

Comment:
This is pure optimization.  Using the `string()` method is a lot faster when you're mostly just dealing with the columns as strings.